src/components/navigation/NavBar.tsx

```diff
@@ -0,0 +1,35 @@
+import { Link, List, ListItem } from "@mui/joy";
+import NextLink from "next/link";
+import { Logo } from "./Logo";
+
+export const NavBar = () => {
+    return (
+        <header>
+            <nav className="flex items-center justify-around flex-wrap bg-blue-500 p-6">
+                <div className="flex items-center flex-shrink-0 text-white mr-6">
+                    <Logo />
+                    <span className="font-semibold text-xl tracking-tight pl-3">
+                        Pocket Assistant
+                    </span>
+                </div>
+
+                <div className="w-full block lg:flex lg:items-center lg:w-auto">
```

Comment:
This should be a `<nav />` element. And it is pretty common to have a `<ul />` inside it with the `<li />` as items of the navigation